README.md

```diff
@@ -1,22 +1,26 @@
+Azure customers find it extremely difficult to manage the lifecycle of a _collection_ of resources – while it’s easy to deploy resources together as a group, after the deployment finishes there is no single way to relate those resources together to manage their lifecycle. Infrastructure deployed in Azure may span multiple resource groups, subscriptions and even tenants. Deployment Stacks will make it easy to manage the lifecycle of a collection resources that work together to create a solution.
 
-A "deploymentStack" is a grouping concept that allows for lifecycle operations to be performed on the defined group.
+A "deploymentStack" is a grouping concept that allows for lifecycle operations to be performed on the defined group of resources. While it is very similar to a traditional [Microsoft.Resources/deployments](https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/deployments?tabs=json), a deploymentStack is a reusable resource that can help you manage the resources your deployment creates. Any resource deployed using a deploymentStack is _managed_ by that deploymentStack, and subsequent updates to the deploymentStack, combined with a `UpdateBehavior` will allow you to control the lifecycle of the resources managed by the deploymentStack. When a deploymentStack is updated, all previously managed resources are replaced with the resources defined in the template used to update the stack. The UpdateBehavior property of the deploymentStack allows you to do the following: 
+
+* `DetachResources`: Keep the previously managed resources in Azure but remove them for the list of the stack's managedResources.
+* `PurgeResources`: Delete the previously managed resources so that they no longer exist in Azure.
 
 ## Known limitations
 
 There are the known limitations with the private preview
 
-- Lock/unlock resources is unavailable. Lock/unlock performs an operation on the set of resources to prevent (Lock) or enable (Unlock) changes.
-- What-if is unavailable. What-if allows for evaluating changes before deploying.
-- The purge mode of a stack does not purge resourceGroups, subscriptionAliases, or managementGroups that are created by the stack.
-- Can't create managed group level stacks.
-- It is not recommended to use stack in production environment because this release returns secured strings, secured objects, and the template.
-- Bicep support not available.
-- Portal support not available.
+- It is not recommended to use deploymentStacks in production environment since a deploymentStack doesn not obey secured strings or objects.
```

Comment:
What does it mean to say "secureStrings or objects" aren't obeyed?

Also we should separate the "don't use in prod" and the "secure param" bullet points

Reply:
Done, i don't think if fully understand securestring and secureobjects, but have made a clarifying change